psalm-baseline.xml

```diff
@@ -509,6 +515,9 @@
     </MixedAssignment>
   </file>
   <file src="src/Operation/FindOneAndReplace.php">
+    <MixedArgument>
+      <code><![CDATA[$options['codec'] ?? null]]></code>
+    </MixedArgument>
```

Comment:
Not sure why this errors in that one specific file - it works in all other files where we have the same checks on the codec option and then use it in a typed context 🤷

Reply:
I couldn't reproduce.
https://psalm.dev/r/609a35bce9